integration-tests/spring-data-jpa/src/test/java/io/quarkus/it/spring/data/jpa/BookResourceTest.java

```diff
@@ -132,4 +136,15 @@ void testCustomFindPublicationYearObject() {
                 .statusCode(200)
                 .body(is("2011"));
     }
+
+    @Test
+    void testEnsureFieldPageableIsSerialized() {
+        Response response = given()
+                .accept("application/json")
+                .queryParam("size", "2")
+                .queryParam("page", "1")
+                .when().get("/book/paged");
+        Assertions.assertEquals(200, response.statusCode());
+        assertThat(response.body().jsonPath().getString("pageable")).contains("paged:true, unpaged:false");
```

Comment:
IIRC, there's a `containsInAnyOrder()` or similar and that's what you need to use except if the order is meaningful.

Reply:
Ah, yes. I had the intention but as it didn't fail locally, I forgot about it.